index.js

```diff
@@ -107,6 +107,10 @@ if(process.env.GOOGLE_PROJECT_ID && process.env.GOOGLE_API_KEY){
         region: process.env.GOOGLE_DEFAULT_REGION || 'us-east1'
     };
 }
+if(process.env.GOOGLE_APPLICATION_CREDENTIALS){
+    GoogleConfig = require(process.env.GOOGLE_APPLICATION_CREDENTIALS)
+    GoogleConfig.project = GoogleConfig.project_id;
```

Comment:
where is the `GoogleConfig.project_id` set? the block if above?
if that's the case it's better to check the `GOOGLE_PROJECT_ID` env in this block and use it instead.

with current approach, you'll have to set GOOGLE_PROJECT_ID, GOOGLE_API_KEY & GOOGLE_APPLICATION_CREDENTIALS to get it properly configured

Reply:
The thing is, the service needs to load `project` in the internal logic but the default service JSON file from GCP only provides `project_id`. This change is to make that we only have to provide the file without the need to add another configuration. Please notice that it use separate `if` to check separate way to configure GoogleConfig

Reply:
ok, noted